src/main/resources/Templates/fragments/navbar.html

```diff
@@ -0,0 +1,8 @@
+<div th:fragment="navbar">
+    <nav>
+        <ul>
+            <li><a href="/">Home</a></li>
+            <li><a href="addRecipe">Add Recipe</a></li>
+        </ul>
+    </nav>
+</div>
```

Comment:
# Change 2

To help with future scalability, I have moved the navbar into a separate file that is reusable.

Since the navbar is used on many places of the site, this change should simplify the process of updating it in the future.

Note: to include it in any new files, do the following (I have added it to your existing pages in this PR):
```html
<div th:replace="fragments/navbar :: navbar"></div>
```